src/CommunityToolkit.Maui/Converters/BaseConverterOneWay.shared.cs

```diff
@@ -13,13 +13,13 @@ namespace CommunityToolkit.Maui.Converters;
 public abstract class BaseConverterOneWay<TFrom, TTo> : ValueConverterExtension, ICommunityToolkitValueConverter
```

Comment:
No, because it relies on `Microsoft.Maui.Controls`.

`BaseConverterOneWay` implements `ICommunityToolkitValueConverter` which implements `Microsoft.Maui.Controls.IValueConterter`